mdn/dom-examples
# Fixing channel messaging type error

#### Changes

- Guards against the event ports when receiving a message from the main page.

#### Testing Procedure

1. Visit `/dom-examples/channel-messaging-basic/`
2. Instead of seeing this: 
![Screen Shot 2023-09-19 at 4 46 49 PM](https://github.com/mdn/dom-examples/assets/11777364/5eb5c238-6c3c-4e81-a4ad-4c0cb1bcccf1)
3. You should see this: 
![Screen Shot 2023-09-19 at 4 46 36 PM](https://github.com/mdn/dom-examples/assets/11777364/4b5db700-d1e5-4f36-b1a4-814040a5fe39)
